tests/cpgrid/addLgrsOnDistributedGrid_test.cpp

```diff
@@ -714,14 +803,48 @@ BOOST_AUTO_TEST_CASE(distributed_lgr_II)
 
         grid.addLgrsUpdateLeafView(cells_per_dim_vec, startIJK_vec, endIJK_vec, lgr_name_vec);
         refinePatch_and_check(grid, cells_per_dim_vec, startIJK_vec, endIJK_vec, lgr_name_vec);
+
+        // Check global id is not duplicated for points for each LGR
+        // LGR1 dim 6x2x2 -> 7x3x3 = 63 points
+        std::vector<int> local_point_ids;
+        local_point_ids.reserve(63); // expected_point_ids in LGR1
+        for (const auto& element : elements(grid.levelGridView(1))) {
```

Comment:
What is the reason why we do not iterate over vertices directly?

Reply:
Good point!